src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -9,107 +9,113 @@
 import java.util.List;
 import java.util.Optional;
 import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.exception.DataInsertException;
 import mate.jdbc.lib.exception.DataProcessingException;
 import mate.jdbc.model.Manufacturer;
 import mate.jdbc.util.ConnectionUtil;
 
 @Dao
 public class ManufacturerDaoImpl implements ManufacturerDao {
+
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country) "
-                + "VALUES (?, ?)";
+        String insertManufacturerRequest = "INSERT INTO manufacturers (name,country) values(?,?);";
         try (Connection connection = ConnectionUtil.getConnection();
-                PreparedStatement preparedStatement
-                        = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
-            preparedStatement.setString(1, manufacturer.getName());
-            preparedStatement.setString(2, manufacturer.getCountry());
-            preparedStatement.executeUpdate();
-            ResultSet resultSet = preparedStatement.getGeneratedKeys();
-            if (resultSet.next()) {
-                manufacturer.setId(resultSet.getObject(1, Long.class));
+                PreparedStatement createManufacturerStatement =
```

Comment:
same here createStatement is enough And check everywhere

Reply:
done